spock-specs/src/test/groovy/org/spockframework/smoke/ast/AstSpec.groovy

```diff
@@ -11,8 +11,8 @@ import spock.util.Show
 
 import org.codehaus.groovy.control.CompilePhase
 
+@Snapshot(extension = 'groovy')
```

Comment:
The downside of the type-level annotation is that, it doesn't support two instances with different extensions.